### ISSUE TITLE
[feat] integrate occular 

### ISSUE THREAD

**Issue by LES-11** [Scratch-Client-4/itchy-ionic#62]
Integrate occular status from https://ocular.jeffalo.net/
This is the same that is integrated into scratchaddons
View screenshots below
(image 1) shows setting your status
(image 1) shows how scratchaddons displays it on profiles

API here https://ocular.jeffalo.net/docs/api

image 1
![image](https://user-images.githubusercontent.com/75516401/161679909-2be539ee-8dba-46a5-812c-77643e06f6e0.png)
image 2
![image](https://user-images.githubusercontent.com/75516401/161680325-d3aa9560-e509-4517-9ed3-991a77d9fe4c.png)

**Comment by micahlt** [comment]
Fun idea!  Backlog for now though :)

**Comment by locness3** [comment]
Imagine if we had integrated ScratchNetwork back in the days

**Comment by micahlt** [comment]
> Imagine if we had integrated ScratchNetwork back in the days

What's ScratchNetwork?  I must not have been around for that.

**Comment by locness3** [comment]
ScratchNetwork was a website that somewhat integrated with Scratch. I believe it was originally meant for people to promote and manage shops, and it had a currency named PushPoints. You should read the [announcement post](https://scratch.mit.edu/discuss/topic/297417/) (you can view the images if you search for their URL on the Wayback Machine) and [a post about PushPoints](https://scratch.mit.edu/discuss/topic/300538/) on Discuss Scratch. There's [a snapshot of it](https://web.archive.org/web/20180609222321/https://edxt.net/ed/scratchnetwork/#) available, where you can see that the Scratch Team didn't like it because it was "a self-moderated service".

**Comment by micahlt** [comment]
That looks like fun lol

**Comment by locness3** [comment]
Hoping that our project doesn't end like this one.

**Comment by micahlt** [comment]
Nah, that's more of an attempt to build a community.  Itchy is just a client - I don't see a lot of similarity.
